specification/src/chapter2.adoc

```diff
@@ -155,8 +156,59 @@ break security guarantees, either directly or indirectly. For example:
 
 | CAT_NNN
 | Invasive subsystems MUST be controlled, or moderated, by a RoT.
+
+|===
+
+==== Event counters
+
+Event counters are commonly used for performance management and resource allocation. 
+
+However, they can also pose a security risk. For example, one workload measuring an operation in a different workload, or an operation by higher privilege software, could be able to reveal assets used in those operations.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#
+| Requirement
+
+| CAT_NNN
+| Lower privilege software MUST NOT be able to measure higher privilege software.
+
+| CAT_NNN
+| Software in one domain MUST NOT be able to measure software in a different domain, without consent.
```

Comment:
monitor instead of measure?

Reply:
"monitor" in this case would be too broad of a word to use in this context as it takes into account side-channels. Would "architecturally monitor" be too verbose?

Reply:
I think better to be specific - how about - 

Software in one domain MUST NOT be able to monitor software in a different domain using architectural performance monitoring events/counters, without consent.

Reply:
I changed to "Monitor" throughout the document.

These requirements are all in the context of Event counting, or architectural platform QoS. So taken in context of the sections they are in they should be clear enough without repeating context info everywhere?

Reply:
I did the same update in the surrounding text as well.